cloudconfig/userdatacfg_unix.go

```diff
@@ -166,6 +166,9 @@ func (w *unixConfigure) ConfigureBasic() error {
 			}
 			w.addCleanShutdownJob(initSystem)
 		}
+		// We disallow "do-release-upgrade" to run on Juju because we do not want to break mongodb.
+		// - https://bugs.launchpad.net/juju/+bug/1881218
+		w.conf.AddRunCmd("sed  -i '/Prompt=/ s/=.*/=never/' /etc/update-manager/release-upgrades")
```

Comment:
this only works if there is already a release-upgrades file and it has a prompt entry
i guess that's ok as it is what's expected to be there

Reply:
also though, this will be done on all machines right?
we want to limit it to controller machines only
i bet if you did a juju add-machine and checked, this would be there

Reply:
That's right! the `juju upgrade-series` just use the `do-release-upgrade` as well.